content/linked-terms.js

```diff
@@ -1,14 +1,22 @@
 module.exports = [
+  {
+    matches: 'dvc.yaml',
+    url: '/doc/user-guide/project-structure/pipelines-files'
+  },
+  {
+    matches: 'dvc.lock',
+    url: '/doc/user-guide/creating-pipelines#dvclock-file'
+  },
   {
     matches: '.dvc',
     url: '/doc/user-guide/project-structure/dvc-files'
   },
   {
-    matches: 'dvc.yaml',
-    url: '/doc/user-guide/project-structure/pipelines-files'
+    matches: '.dvcignore',
+    url: '/doc/user-guide/project-structure/dvcignore-files'
```

Comment:
Hi @rogermparent 👋 

For some reason this new autolink is not being picked up in https://dvc-landing-2-0-pipelin-xkejno.herokuapp.com/doc/user-guide/project-structure.

It works everywhere in https://dvc-landing-2-0-pipelin-xkejno.herokuapp.com/doc/command-reference/add and https://dvc-landing-2-0-pipelin-xkejno.herokuapp.com/doc/command-reference/check-ignore.

I notices that if the bullet doesn't start with `` `.dvcignore` `` then it works. But that's not a problem for the previous bullets which start with `` `dvc.yaml` `` and `` `.dvc` `` 🤷

Reply:
Looks like it's being picked up to me. Maybe it was a cache issue?

Reply:
Yep. Thanks @rogermparent

Reply:
Well, it still happens on my local even after `rm -rf .cache/` but I see it's OK in the review app 🤷